package.json

```diff
@@ -42,7 +42,7 @@
     "test-node-cli-mode": "bash < scripts/test-node-cli-mode.sh",
     "test-std-in": "bash < scripts/test-std-in.sh",
     "test-react": "jest scripts/test-react",
-    "test": "yarn test-residual && yarn test-serializer && yarn test-sourcemaps && yarn test-error-handler && yarn test-std-in && yarn test-test262 && yarn test-react && yarn test-internal",
+    "test": "yarn test-residual && yarn test-serializer && yarn test-sourcemaps && yarn test-error-handler && yarn test-std-in && yarn test-test262 && yarn test-internal",
```

Comment:
Is removing test-react intentional? If not, keep it!

Reply:
It was intentional, but it has since been fixed. I'll revert.